include/output.hxx

```diff
@@ -195,32 +195,32 @@ public:
   };
 
   Output *base;
-  bool enabled;
 
 private:
+  bool enabled;
   bool base_is_cond;
-
-private:
 };
 
 /// Catch stream outputs to DummyOutput objects. This is so that
 /// statements like
 ///    output_debug << "debug message";
 /// compile but have no effect if DEBUG_ENABLED is false
-template <typename T> DummyOutput &operator<<(DummyOutput &out, T const &t) {
+template <typename T> DummyOutput &operator<<(DummyOutput &out, T const &UNUSED(t)) {
   return out;
 }
 
-template <typename T> DummyOutput &operator<<(DummyOutput &out, const T *t) {
+template <typename T> DummyOutput &operator<<(DummyOutput &out, const T *UNUSED(t)) {
   return out;
 }
 
-inline DummyOutput &operator<<(DummyOutput &out, std::ostream &(*pf)(std::ostream &)) {
+// Function pointer so we can apply unused macro to pf in function below
+using stream_manipulator = std::ostream &(*)(std::ostream &);
+
+inline DummyOutput &operator<<(DummyOutput &out, stream_manipulator *UNUSED(pf)) {
```

Comment:
This is now broken.
Not quite sure why, but I think it is one * more then before.